src/internal/utils/date-time/__tests__/format-date-localized.test.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I ran this test locally without the changes in `format-date-localized` and it also passed.

Reply:
oh right, because I reproduced this problem locally after changing my system timezone to pacific